src/as-validator.c

```diff
@@ -893,8 +893,13 @@ as_validator_check_appear_once (AsValidator *validator, xmlNode *node, GHashTabl
 	lang = as_xml_get_prop_value (node, "lang");
 	if (lang == NULL)
```

Comment:
style: This *does* need braces now, as the following `else` branch also has them